integration/benchnet2/automate/templates/helm-values-all-nodes.yml

```
@@ -0,0 +1,385 @@
+        - --datadir=/data/protocol
+        - --secretsdir=/data/secret
+        - --loglevel=INFO
+        - --bind=0.0.0.0:3569
+        - --profiler-enabled=false
+        - --profile-uploader-enabled=false
+        - --tracer-enabled=false
+        - --profiler-dir=/profiler
+        - --profiler-interval=2m
+        - --nodeid={{$val.node_id}}
+        - --triedir=/trie
+        - --rpc-addr=0.0.0.0:9000
+        - --cadence-tracing=false
+        - --extensive-tracing=false
+      env:
+        - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+          value: http://tempo:4317
+        - name: OTEL_EXPORTER_OTLP_TRACES_INSECURE
+          value: "true"
+        - name: OTEL_RESOURCE_ATTRIBUTES
+          value: network
+        - name: BINSTAT_ENABLE
+          value: ""
+        - name: BINSTAT_LEN_WHAT
+          value: ""
+        - name: BINSTAT_DMP_NAME
+          value: ""
+        - name: BINSTAT_DMP_PATH
+          value: ""
+      image: gcr.io/flow-container-registry/execution:v0.27.6
+      nodeId: {{$val.node_id}}
+{{end}}{{end}}
+verification:
+  defaults:
+    imagePullPolicy: Always
+    containerPorts:
+      - name: ptp
+        containerPort: 3569
+      - name: grpc
+        containerPort: 9000
+      - name: secure-grpc
+        containerPort: 9001
+      - name: admin
+        containerPort: 9002
+    env: []
+    resources:
+      requests:
+        cpu: "200m"
+        memory: "512Mi"
+      limits:
+        cpu: "800m"
+        memory: "10Gi"
+    servicePorts:
+      - name: ptp
+        protocol: TCP
+        port: 3569
+        targetPort: ptp
+      - name: grpc
+        protocol: TCP
+        port: 9000
+        targetPort: grpc
+      - name: secure-grpc
+        protocol: TCP
+        port: 9001
+        targetPort: secure-grpc
+      - name: admin
+        protocol: TCP
+        port: 9002
+        targetPort: admin
+    storage: 1G
+  nodes:
+{{range $val := .}}{{if eq ($val.role) ("verification")}}
+    {{$val.name}}:
+      args:
+        - --bootstrapdir=/bootstrap
+        - --datadir=/data/protocol
+        - --secretsdir=/data/secret
+        - --loglevel=INFO
+        - --bind=0.0.0.0:3569
+        - --profiler-enabled=false
+        - --profile-uploader-enabled=false
+        - --tracer-enabled=false
+        - --profiler-dir=/profiler
+        - --profiler-interval=2m
+        - --nodeid={{$val.node_id}}
+        - --chunk-alpha=1
+      env:
+        - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
+          value: http://tempo:4317
+        - name: OTEL_EXPORTER_OTLP_TRACES_INSECURE
+          value: "true"
+        - name: OTEL_RESOURCE_ATTRIBUTES
+          value: network
+        - name: BINSTAT_ENABLE
+          value: ""
+        - name: BINSTAT_LEN_WHAT
+          value: ""
+        - name: BINSTAT_DMP_NAME
+          value: ""
+        - name: BINSTAT_DMP_PATH
+          value: ""
+      image: gcr.io/flow-container-registry/verification:v0.27.6
+      nodeId: {{$val.node_id}}
+{{end}}{{end}}
```

If we alter this line, we can remove the need to strip whitespace from the end of the file.

```suggestion
{{- end -}}{{- end -}}
```

---

Added in cbc51ffdab0c5e33c9482d4efd48400eb58ee3aa but I was getting a lot of failing test cases if I removed the white space trimming so I kept that in. There are ways in which the yaml files are saved by the IDE that automatically put in a new line at the end and this makes really problematic for unit testing if I take out the white space trimming.